### ISSUE TITLE
Suggestion: Make success and failure markers more concise

### ISSUE THREAD

**Issue by Gert-dev** [yacut/tester#21]
Another idea (I'm prolific with idea's and suggestions today 😄) I had was to replace the current `passed` and `failed` markers with more concise symbols:

![test](https://user-images.githubusercontent.com/24809863/27448095-fe3725e2-5772-11e7-96eb-327e05f38f64.png)

The reasoning is that, even though the green and red colors are very useful to quickly see what the status is, having a lot of colored content makes the results pane rather heavy on the eyes. All I did here was use the Atom inspector to replace `success` with the unicode check mark ✓ and `failed` with the unicode cross ⨯ (or rather it's a vector cross product symbol, I think, but it looked rather well) and finally decrease the `width` and `min-width` of the `.tester-messages .tester-message-state` class.

What do you think? (Of course it's fine too if you don't agree 😄!)

**Comment by yacut** [comment]
I think that is great idea. But I would like to get both view options: icon or text.
Could you please create a pull request for this?

**Comment by Gert-dev** [comment]
My JavaScript-knowledge (especially with regard to modern libraries) isn't that great, but I managed to get in the different styling without the configurability. This should give you a head start:

```patch
diff --git a/lib/views/ResultView.js b/lib/views/ResultView.js
index 3e63005..89200bd 100644
--- a/lib/views/ResultView.js
+++ b/lib/views/ResultView.js
@@ -105,7 +105,7 @@ export default class ResultView {
         <div class='tester-messages' ref='messages' messages={messages}>
           <div class='tester-message-header inline-block'>
               <div class='tester-message-cell tester-header tester-message-state inline-block'
-                onclick={this.handleSortByClick.bind(this, 'state')} >State
+                onclick={this.handleSortByClick.bind(this, 'state')} >✓
                 <span class={`sort-indicator icon${sortKey === 'state' ? desc ? ' desc icon-triangle-down' : ' asc icon-triangle-up' : ''}`}
                   ref='header-state'></span>
               </div>
@@ -145,7 +145,12 @@ export default class ResultView {
                         'highlight-error' : message.state === 'skipped' ?
                         'highlight-warning' : message.state === 'passed' ?
                         'highlight-success' : 'unknown'
-                      }>{message.state || 'unknown'}</span>
+                      }>{
+                        message.state === 'failed' ?
+                        '⨯' : message.state === 'skipped' ?
+                        '-' : message.state === 'passed' ?
+                        '✓' : '?'
+                      }</span>
                     </div>
                     <div class='tester-message-cell tester-message-duration inline-block'>{message.duration || 0}ms</div>
                     <div class='tester-message-cell tester-message-title inline-block'>{message.title || ''}</div>
diff --git a/styles/tester.less b/styles/tester.less
index 39e1f3f..fe5c912 100644
--- a/styles/tester.less
+++ b/styles/tester.less
@@ -180,8 +180,11 @@ atom-text-editor.editor .tester-highlight, .tester-highlight{
     text-overflow: inherit;
   }
   .tester-message-state {
-    min-width: 5rem;
-    width: 5%;
+    min-width: 2.5rem;
+    width: 4%;
+  }
+  .tester-message-row .tester-message-state {
+    text-align: center;
   }
   .tester-message-duration {
     min-width: 5rem;
```

**Comment by yacut** [comment]
Released in 1.4.0
